### ISSUE TITLE
Was wondering if A3 is compatible

### ISSUE THREAD

**Issue by robbiedood** [NickL77/RPLidar_Hector_SLAM#9]
Dear NickL77,

Thanks for the great work to make RPLidar running with Hector SLAM on ROS. 
The latest RPLidar A3M1 is launched, so I was wondering if you have a plan to update the github, make it compatible with A3.

Thank you !

**Comment by MI-Hussain** [comment]
Yes it is working with RPLidar A3. Just replace the rplidar ros drivers, with this https://github.com/robopeak/rplidar_ros.git.

this will perfectly work.
